packages/launchpad/src/components/code/FileRow.vue

```diff
@@ -98,6 +100,9 @@ const props = defineProps<{
   fileExtension: string
 }>()
 
+const LEARN_MORE_URL = 'https://on.cypress.io/guides/configuration'
```

Comment:
nit: define this constant at the top of the file under the imports

Reply:
fixed in a4500825116f1c10fc5814711ff31b4769d9173e